multimodal/tests/unittests/others_2/test_backward_compatibility.py

```diff
@@ -43,3 +44,32 @@ def test_load_old_checkpoint(cls):
         time_limit=10,
         hyperparameters={"optimization.top_k_average_method": "uniform_soup", "env.num_gpus": 1},
     )
+
+
+def test_v0_6_2_checkpoint_timm_image():
```

Comment:
Can I ask what this unit test is for? Thank you

Reply:
Current checkpoints also have “timm_image.image_size”. However, before this PR, we will simply ignore this flag. This PR correctly implements the “timm_image.image_size” and also changed the default value from 224 to “null”. We need to test the backward compatibility (i.e. we are able to load the checkpoints saved with autogluon.multimodal<=0.6.2).